simtools/util/tests.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+CONFIG_FILE_MSG = "Configuration file is not available"
+
+DB_CONNECTION_MSG = "There is not connection with DB"
```

Comment:
Change to "Connection with the DB is not available".

Reply:
Done